juno-migration/9481749/project_juno.yaml

```diff
@@ -0,0 +1,97 @@
+specVersion: 1.0.0
+name: juno-dictionary
+version: 1.0.0
+runner:
+  node:
+    name: "@subql/node-cosmos"
+    version: ">=3.0.0"
+    options:
+      unfinalizedBlocks: false
+      historical: false
+  query:
+    name: "@subql/query"
+    version: "*"
+description: A SubQuery project provide dictionary data & service for the Terra network
+schema:
+  file: ./schema.graphql
+
+parent:
+  reference: QmT9M7AZSWgh8KrHUS2Q5kGPSy2FTankgnoPd8KKA9J7nT
```

Comment:
Hi @web3cdnservices , thank you for the pr. Everything looks good, I think expect this block should be 9481750, this is the block where start throw error

9481749 will throw error

```
<Project> ERROR Unable to upgrade project. Cannot rewind to block 9481748 without historical indexing enabled. 
```

Reply:
Thanks. Changed to `9481750`